packages/sanity/src/core/store/_legacy/user/userStore.ts

```diff
@@ -40,7 +40,6 @@ export function createUserStore({client: _client, currentUser}: UserStoreOptions
     async (userIds) => {
       const value = await client.request<(User | null)[]>({
         uri: `/users/${userIds.join(',')}`,
-        withCredentials: true,
```

Comment:
@ricokahler  Can you confirm this to be safe? Any idea why we passed `withCredentials: true` here explicitly? (this should have implicitly been the case unless a client not based on the source's `getClient`/`useClient` was used)